src/pages/contributors/adrien-lecharpentier.adoc

```diff
@@ -0,0 +1,84 @@
+= Adrien Lecharpentier
+:page-name: Adrien Lecharpentier
+:page-linkedin:
+:page-twitter: 
+:page-github: alecharp
+:page-email:
+:page-image: avatar/adrien-lecharpentier.jpg
+:page-pronouns: He/Him/His
+:page-location: Nantes, France
+:page-firstcommit: 2009
+:page-datepublished: 2024-10-08
+:page-featured: true
+:page-intro: Adrien is a software engineer currently based in Nantes, France. While originally born in Normandy, Adrien called Brittany and Paris home while working on his education. Thanks to having teachers for parents, Adrien was always encouraged in both mental and physical pursuits, enjoying activities such as football, basketball, Aikido, and handball. After getting his baccalaureate, Adrien studied IT sciences, such as networking and the more physical layers, for two years before diving into IT programming. After getting an additional diploma for these studies, Adrien found an engineering school in Nantes, but found their way of teaching wasn't ideal for his learning. This led him to find another school in Paris where his time was split between school and on-site work. Being in this structure allowed him to focus more on programming, gain industry exposure, and better understand industry standards.
+
+While he tried to keep up with physical activities, Paris made it a bit tough. However, it was at this point that he discovered hiking with his partner. It was also during this time that Adrien got into photography. Throughout his life, he has always been tinkering with electronics, which eventually led to 3D printing. Adrien has found that 3D printing allows him to discover and try new things when he comes across an interesting project. It also provides him with space to make things that solve problems, even creating some multi-problem-solving tools. Thanks to 3D printing, being able to customize and create solutions for any problem encountered has pushed his creativity and problem-solving skills further than even before. Recently, Adrien also started woodworking with his father, using his eye for detail and love for making things to do something more analog. While there is a difference in the medium, his desires to create and problem solve transfer directly over to his new hobby.
```

Comment:
It's just about temporality here. Electronics since teenager, IT since high school, woodworking since mid-20's and more "recently" 3D printing ;)

```suggestion
While he tried to keep up with physical activities, Paris made it a bit tough. However, it was at this point that he discovered hiking with his partner. It was also during this time that Adrien got into photography. Adrien also started woodworking with his father, a carpenter by training, psychologist by trade, using his eye for detail and love for making things to do something more analog. Throughout his life, he has always been tinkering with electronics, which eventually led to 3D printing. While there is a difference in the medium, his desires to create and problem solve transfer directly over to his new hobby. Adrien has found that 3D printing allows him to discover and try new things when he comes across an interesting project. It also provides him with space to make things that solve problems, even creating some multi-problem-solving tools. Thanks to 3D printing, being able to customize and create solutions for any problem encountered has pushed his creativity and problem-solving skills further than even before.
```

Reply:
Thanks @alecharp, I've updated the intro accordingly, to make it make sense in terms of timeline